server/test/resolve.cpp

```diff
@@ -1211,3 +1211,59 @@ TEST_CASE("Resolve by hosted by")
         FAIL(ex.what());
     }
 }
+
+TEST_CASE("Resolve by Group")
+{
+    try {
+        fty::SampleDb db(R"(
+            items:
+                - type : Datacenter
+                  name : datacenter
+                  items:
+                    - type     : Server
```

Comment:
Best will be to add another asset like "datacenter", and make sure that this one is not on the group, and also in group linked to the group.